### PR TITLE
Release v0.1.1

### LINKED ISSUE #5: update the format of output txt
include "#" in the first few lines

### PR DESCRIPTION
**Pull request by MLK1998**
ENH: add xyz file support (Automatic detection input), radial distribution function, and output optimization.
DEP: deprecate ./pycsro/ folder.

This PR resolves #5 , #6 , and #8

### CODE REVIEW

**Comment by MLK1998** [comment]
@dragonyanglong

File: src/pycsro/io.py [hunk 1]
```diff
@@ -37,6 +38,28 @@ def get_config(atoms):
...
+        file_name (str): The absolute path of input file.
+
+    Returns:
+        file_name (str): The absolute path of input file.
+        initial_file_name (str): The initial absolute path of input file.
+    """
+    initial_file_name = file_name
+    try:
+        atoms = read(file_name)
+    except Exception:
```
Comment (dragonyanglong) [inline src/pycsro/io.py:55]: 这样写可以包括全部的报错吗？测试过一些吗？

File: src/pycsro/neighbor.py [hunk 2]
```diff
@@ -163,3 +171,209 @@ def plot_partial_neighbors(cutoff, neighbors_1, neighbors_2, ele, neighbors_ele,
...
+        if plot_save:
+            plt.savefig('rdf', dpi=300)
+            plt.show()
+        else:
+            plt.show()
+    return save_plot_data
+
+
+
+def plot_for_nieighbors(partial_neighbors, cutoff2, neighbors, neighbors_2,
```
Comment (dragonyanglong) [inline src/pycsro/neighbor.py:345]: fix typo

**Review by dragonyanglong — COMMENTED** [review]
@MLK1998  please fix per my comments.

And, please add more testing cases.

**Comment by dragonyanglong** [comment]
Then you can mentioned the issues related with your PR in the comment. Such as "This PR resolves #6", it will automatically link with the No. 6 issue.

**Comment by dragonyanglong** [comment]
please update README, for the new version of example, and outputs.

File: README.md [hunk 3]
```diff
@@ -62,12 +63,16 @@ run_pycsro_pmsro(ion1, cutoff1, file_name, cutoff2, save_name, skip_distance, pl
...
-2. Supported input file formats incllude `CIF`, `POSCAR`.
+2. Supported input file formats incllude `CIF`, `POSCAR`, `xyz`, and more.
 3. For compounds, cations and anions should be placed in different ion groups, and calculated separately.
 4. You can try a larger cutoff value at the first time, and then adjust the cutoff value to the trough of the neighbor distribution.
 5. The second shell was defined as cutoff1 to cutoff2.
+6. Noremally, the program automatically detected the xyz input file without specifying an additional xyz parameter. This parameter can be manually selected when the automatic judgment fails.
+7. When the input file is in xyz format, the radial distribution function will be switched to the local neighbor statistics function.
 
 
 Example
```
Comment (dragonyanglong) [inline README.md:78]: please update.

File: README.md [hunk 4]
```diff
@@ -62,12 +63,16 @@ run_pycsro_pmsro(ion1, cutoff1, file_name, cutoff2, save_name, skip_distance, pl
...
+- `xyz`: Whether use the xyz mode to read input file. (Default: No, range: Yes or No, type: str)
+
 Attention: 
 1. The calculation requires the user to input parameters of `ion1`, `cutoff1`, and `file_name`.
-2. Supported input file formats incllude `CIF`, `POSCAR`.
+2. Supported input file formats incllude `CIF`, `POSCAR`, `xyz`, and more.
 3. For compounds, cations and anions should be placed in different ion groups, and calculated separately.
 4. You can try a larger cutoff value at the first time, and then adjust the cutoff value to the trough of the neighbor distribution.
 5. The second shell was defined as cutoff1 to cutoff2.
+6. Noremally, the program automatically detected the xyz input file without specifying an additional xyz parameter. This parameter can be manually selected when the automatic judgment fails.
```
Comment (dragonyanglong) [inline README.md:74]: typo.

File: README.md [hunk 5]
```diff
@@ -20,6 +20,7 @@ Required Dependencies:
 * numpy
 * scipy
 * ase
+* pymatgen
```
Comment (dragonyanglong) [inline README.md:23]: can pymatgen be installed in pip? Did you test in a new conda environment?

**Review by dragonyanglong — COMMENTED** [review]
@MLK1998

File: src/pycsro/io.py (hunk 1 above)
Comment (MLK1998) [reply src/pycsro/io.py:55]: 可以的包含全部的报错，我手头读不出来的CIF，这样写是可以运行的。

Comment (dragonyanglong) [reply src/pycsro/io.py:55]: 好的，你点下“resolve conversation"，所有的comments完成后，再@我下，我来review 和merge

Comment (dragonyanglong) [reply src/pycsro/io.py:55]: 需要修改的地方，你直接正常git push，  github就会自动更新进现有的这个PR,不需要重新开PR

File: README.md (hunk 5 above)
Comment (MLK1998) [reply README.md:23]: Pymatgen can be installed in pip. The program has been tested in a new conda environment.

File: README.md (hunk 4 above)
Comment (MLK1998) [reply README.md:74]: fixed

File: src/pycsro/neighbor.py (hunk 2 above)
Comment (MLK1998) [reply src/pycsro/neighbor.py:345]: fexed

**Comment by MLK1998** [comment]
@dragonyanglong

**Comment by dragonyanglong** [comment]
This PR resolves #5 #6 #8 issues.

**Comment by MLK1998** [comment]
@dragonyanglong

**Comment by dragonyanglong** [comment]
merged. close #5 #6 #8 issues.

**Comment by dragonyanglong** [comment]
resolve #5

**Pull request merged**